ci-operator/config/openshift-priv/operator-framework-operator-controller/openshift-priv-operator-framework-operator-controller-release-4.19.yaml

```
@@ -47,13 +47,30 @@ images:
       as:
       - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.22-openshift-4.18
   to: e2e-test-registry
+- dockerfile_path: openshift/catalogd.Dockerfile
+  inputs:
+    ocp_4.19_base-rhel9:
+      as:
+        - registry.ci.openshift.org/ocp/4.19:base-rhel9
+    ocp_builder_rhel-9-golang-1.23-openshift-4.19:
+      as:
+        - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.23-openshift-4.19
+  to: olm-catalogd
+- context_dir: catalogd/testdata/catalogs
+  dockerfile_path: test-catalog.Dockerfile
+  from: src
+  to: test-catalog
 promotion:
   to:
   - disabled: true
     excluded_images:
     - e2e-test-registry
     name: 4.19-priv
     namespace: ocp-private
+  - excluded_images:
+    - test-catalog
+    name: 4.19-priv
+    namespace: ocp-private
```

Similar change here:
```suggestion
promotion:
  to:
  - disabled: true
    excluded_images:
    - e2e-test-registry
    - test-catalog
    name: 4.19-priv
    namespace: ocp-private
```